### ISSUE TITLE
module-getter rule false positive

### ISSUE THREAD

**Issue by PatrikElfstrom** [EmmanuelDemey/eslint-plugin-angular#592]
This rule is getting triggered for the following code:

```
import { NgZone } from '@angular/core';

const zone = new NgZone({});
zone.run(() => {
    // stuff
});
```

Is this related to #137 ?

**Comment by tpeiffer** [comment]
I thought that this library was intended to be used with AngularJS, not Angular2+.  Does it work for you other than this one place?

That being said, I copied this into my (AngularJS) app and got the same error.  I would agree that it is due to the .run() method getting a false positive.  Looking at the code, it appears that the issue is in rules/utils/false-values.js.  I'd try modifying that file locally to see if you can fix the issue.  Perhaps just adding ", /zone/" to the end of the array will fix it for you.  I know, that is a band-aid and won't work for CI scenarios unless you do what I did and make copies of rules I need modified and part of my CI build copies the modified rules over top of the ones in node_modules.

Seems like this rule could use the ability to configure it and pass a list of items known to fail for the user instead of having them hard coded in the source.

**Comment by PatrikElfstrom** [comment]
It seems to be. I must have glossed over that. 
It does seem to work just fine. I have just disabled a few rules.

Thanks. This is not really a big issue for me. 
I will probably just disable the rule for now and see if I can replace this plugin.

Yes that would seem like a good solution.
